generator/action.go

```diff
@@ -333,16 +164,18 @@ func CreateActionFile(stateFields []*gp.FieldDescriptorProto, customFields []*gp
 	}
 
 	// generate the templates
-	getT := template.Must(template.New("get").Funcs(funcMap).Parse(getTemplate))
-	listT := template.Must(template.New("list").Funcs(funcMap).Parse(listTemplate))
-	resetT := template.Must(template.New("reset").Funcs(funcMap).Parse(resetTemplate))
-	createT := template.Must(template.New("create").Funcs(funcMap).Parse(createTemplate))
-	deleteT := template.Must(template.New("delete").Funcs(funcMap).Parse(deleteTemplate))
-	updateT := template.Must(template.New("update").Funcs(funcMap).Parse(updateTemplate))
-	customT := template.Must(template.New("update").Funcs(funcMap).Parse(customTemplate))
+  importsT := template.Must(template.New("imports").Funcs(funcMap).Parse(this.ActionFile.ImportTemplate))
+	getT := template.Must(template.New("get").Funcs(funcMap).Parse(this.ActionFile.GetTemplate))
+	listT := template.Must(template.New("list").Funcs(funcMap).Parse(this.ActionFile.ListTemplate))
+	resetT := template.Must(template.New("reset").Funcs(funcMap).Parse(this.ActionFile.ResetTemplate))
+	createT := template.Must(template.New("create").Funcs(funcMap).Parse(this.ActionFile.CreateTemplate))
+	deleteT := template.Must(template.New("delete").Funcs(funcMap).Parse(this.ActionFile.DeleteTemplate))
+	updateT := template.Must(template.New("update").Funcs(funcMap).Parse(this.ActionFile.UpdateTemplate))
+	customT := template.Must(template.New("update").Funcs(funcMap).Parse(this.ActionFile.CustomTemplate))
```

Comment:
`template.New("custom")`
Not your mistake, but mine. The name is pretty arbitrary, but should probably match the variable name.